services/app-api/src/resolvers/indexHealthPlanPackages.ts

```diff
@@ -13,38 +13,63 @@ export function indexHealthPlanPackagesResolver(
     store: Store
 ): QueryResolvers['indexHealthPlanPackages'] {
     return async (_parent, _args, context) => {
-        // This resolver is only callable by state users
         const { user, span } = context
         setResolverDetailsOnActiveSpan('fetchHealthPlanPackage', user, span)
-        if (!isStateUser(user)) {
-            const errMsg = 'user not authorized to fetch state data'
-            logError('indexHealthPlanPackages', errMsg)
-            setErrorAttributesOnActiveSpan(errMsg, span)
-            throw new ForbiddenError(errMsg)
-        }
 
-        // fetch from the store
-        const results = await store.findAllHealthPlanPackages(user.state_code)
+        if (isStateUser(user)) {
```

Comment:
We should be able to just change what store function we call based on the user type and still share all the cleanup code. The error handling and the package.map() should be the same.